app/controllers/parks_controller.rb

```diff
@@ -1,4 +1,6 @@
 class ParksController < ApplicationController
+  before_action :authorize_user,
+    except: [:index, :show, :new, :create, :edit, :update]
 
```

Comment:
better to use `only` in a situation like this:
`before_action :authorize_user, only: [:destroy]`